### PR TITLE
mimic: mds: log evicted clients to clog/dbg

### PR DESCRIPTION
**Pull request by ashishkumsingh**
https://tracker.ceph.com/issues/37822

### CODE REVIEW

**Comment by smithfarm** [comment]
jenkins re-test this please

File: src/mds/MDSRank.cc [hunk 1]
```diff
@@ -3203,11 +3203,21 @@ bool MDSRank::evict_client(int64_t session_id,
     return false;
   }
 
+  auto& addr = session->info.inst.addr;
+  {
+    CachedStackStringStream _ss;
```
Comment (smithfarm) [inline src/mds/MDSRank.cc]: In the luminous backport, the fix suggested by @batrick is to use `std::stringstream` instead.

Comment (ashishkumsingh) [reply src/mds/MDSRank.cc]: @smithfarm all changes have been done in the PR as well, thanks for the help.

**Comment by yuriw** [comment]
[wip-yuri3-testing-2019-01-31-2315-mimic](https://trello.com/c/q11X97rN/526-wip-yuri3-testing-2019-01-31-2315-mimic)

**Pull request merged**